.drone.yml

```diff
@@ -0,0 +1,91 @@
+#
+# We are building GCC with make and Clang with ninja, the combinations are more
+# or less arbitrarily chosen. We just want to check that both compilers and both
+# CMake generators work. It's unlikely that a specific generator only breaks
+# with a specific compiler.
+#
+
+workspace:
+  base: /drone
+  path: src/github.com/owncloud/client
+
+branches:
+  - master
+  - 2.4
```

Comment:
There is a 2.5 branch now so i guess it needs to be added there too.

Reply:
Done